### PR TITLE
add acala token config 

### PR DESCRIPTION
**Pull request by lcovar**
TICKET: BG-54526 BG-54166

### CODE REVIEW

**Review by mmcshinsky-bitgo — DISMISSED** [review]

**Comment by mmcshinsky-bitgo** [comment]
@lcovar merge conflicts in need of resolution.

**Comment by lcovar** [comment]
> @lcovar merge conflicts in need of resolution.

fixed, rebuilding.

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -1682,6 +1684,10 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.USDT,
     AccountCoin.DEFAULT_FEATURES
   ),
+  acaToken('aca:AUSD', 'Acala USD', 12, 'AUSD', UnderlyingAsset.AUSD, AccountCoin.DEFAULT_FEATURES),
+  acaToken('aca:LDOT', 'DOT Liquid Staking', 10, 'LDOT', UnderlyingAsset.LDOT, AccountCoin.DEFAULT_FEATURES),
+  tacaToken('taca:AUSD', 'Acala USD', 12, 'AUSD', UnderlyingAsset.AUSD, AccountCoin.DEFAULT_FEATURES),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:1689]:
```suggestion
  tacaToken('taca:AUSD', 'Testnet Acala USD', 12, 'AUSD', UnderlyingAsset.AUSD, AccountCoin.DEFAULT_FEATURES),
```

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -1682,6 +1684,10 @@ export const coins = CoinMap.fromCoins([
...
+  tacaToken('taca:LDOT', 'DOT Liquid Staking', 10, 'LDOT', UnderlyingAsset.LDOT, AccountCoin.DEFAULT_FEATURES),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:1690]:
```suggestion
  tacaToken('taca:LDOT', 'Testnet DOT Liquid Staking', 10, 'LDOT', UnderlyingAsset.LDOT, AccountCoin.DEFAULT_FEATURES),
```

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**